doc/Type/Metamodel/MethodDelegation.pod6

```diff
@@ -0,0 +1,19 @@
+=begin pod
+
+=TITLE role Metamodel::MethodDelegation
+
+=SUBTITLE Metaobject that supports method delegation.
+
+    role Metamodel::MethodDelegation { }
+
+L<Meta model|/language/mop> role for delegating methods.
+
+=head1 Methods
+
+=head2 method delegate_methods_to
+
```

Comment:
I'm not a fan of listing methods without actually documenting them. Then people find them in the search, click on it... and learn absolutely nothing. Better to let them know right in the search that there is no documentation for it yet.

Reply:
Makes sense (this is one of the roles that I was referring to in the description re: "stubbed").

We could change this to a one liner (as I have in `ModuleHOW`) to say something like: "`Metamodel::MethodDelegation` declares `delegate_methods_to`, `delegating_methods_to` and `find_method` (currently undocumented)."

At least that way there's _some_ discoverability?